tasks/deploy.py

```diff
@@ -91,7 +91,7 @@ def _create_stack(ctx: Context, stack_name: str, global_flags: str):
 
 def _deploy(ctx: Context, stack_name: Optional[str], flags: Dict[str, Any], debug: Optional[bool]) -> str:
     stack_name = tool.get_stack_name(stack_name, flags["scenario"])
-    run_wrapper = "aws-vault exec sso-agent-sandbox-account-admin"
+    aws_account = flags["ddinfra:env"][len("aws/"):]
```

Comment:
What do you think about passing `aws_account` as a function parameter instead of parsing `ddinfra:env`?

Reply:
Though about that, but thought also that we have the info in flags. True that flags are for the config map, but I was worried about passing twice the same info

Reply:
We could pass a confimap struct similar to what we have on agent side, and have a function to serialise it to the actual config map dictionary.